magda-web-client/src/UI/Banner.js

```diff
@@ -14,7 +14,24 @@ class Banner extends React.Component {
 
     goBack = event => {
         event.preventDefault();
-        document.cookie = "noPreview=true; path=/";
+
+        if (window.location.hostname === "search.data.gov.au") {
```

Comment:
Do we want to use a cookie lib? e.g. js-cookie?
I can find any issue with your code by looking at it.
The only issue is after a couple tests. my cookie state became a little bit wired:
![image](https://user-images.githubusercontent.com/674387/38653318-8185c9f0-3e4d-11e8-84dd-9888d69db282.png)
And it caused next test failed as it split `"NaN; _vwo_uuid_v2"` into a name

Reply:
Switched to js-cookie 🍪

Reply:
Thank you!
Tested by coding injection (running in chrome js console )
Works fine